ci/check-runtime-benchmarks.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eE -x;
+
+bash -c "while true; do sleep 30; echo \$(date) - running ...; done" &
+PING_LOOP_PID=$!
+trap 'kill $PING_LOOP_PID' ERR 1 2 3 6
+
+# Check if the runtime benchmarks can be compiled
+cd collector/runtime-benchmarks
```

Comment:
While here, is it worth renaming `ci/check-benchmarks.sh` as `ci/check-compile-benchmarks.sh`?

Reply:
Done.